package.json

```diff
@@ -71,7 +71,7 @@
     "html-loader": "^0.5.5",
     "html-loader-jest": "^0.2.1",
     "imports-loader": "^0.8.0",
-    "interactjs": "^1.6.1",
+    "interactjs": "1.6.1",
```

Comment:
I am freezing the interact.js dependency too, because there is a newer interact version available.